app/assets/javascripts/short-form/ShortFormApplicationController.js.coffee

```diff
@@ -578,15 +586,14 @@ ShortFormApplicationController = (
     $scope.eligibilityErrors.length = 0
 
   $scope._respondToHouseholdEligibilityResults = (eligibility, error) ->
-    seniorReqError = $scope.householdDoesNotMeetSeniorRequirements()
-    if eligibility.householdMatch && !seniorReqError
+    if eligibility.householdMatch
       # determine next page of household section
       if ShortFormApplicationService.hasHouseholdPublicHousingQuestion()
         ShortFormNavigationService.goToApplicationPage('dahlia.short-form-application.household-public-housing')
       else
         $scope.checkIfReservedUnits()
     else
-      $scope._determineHouseholdEligibilityErrors(error, seniorReqError)
+      $scope._determineHouseholdEligibilityErrors(error)
```

Comment:
I think with all of these householdEligibilityErrors functions it'd be great to clarify which household eligibility errors? 

I'm confused about the check if reserved units check above, for example.

Reply:
sure I noticed that too, will clarify the function a bit more because there are different HH elig checks